packages/server/modules/core/repositories/tokens.ts

```diff
@@ -74,6 +80,30 @@ export const storePersonalApiTokenFactory =
     return newToken
   }
 
+export const storeEmbedApiTokenFactory =
+  (deps: { db: Knex }): StoreEmbedApiToken =>
+  async (token) => {
+    const [newToken] = await tables.embedApiTokens(deps.db).insert(token).returning('*')
```

Comment:
shouldnt this be in its own repo considering this is all for a new/separate table? our repos/resolvers/services tend to get too big